src/example-types/facet.js

```diff
@@ -175,11 +176,13 @@ module.exports = {
           mapKeywordFilters(node),
         ])
       )
+
       let zeroCountIds = _.difference(
         //when values are numeric values, stringify missedValues to avoid the bug.
-        _.map(_.toString, lostIds),
-        _.map(x => _.toString(x[`${node.field}`]), lostOptions)
+        _.map(F.unless(_.isBoolean, _.toString), lostIds),
+        _.map(({ _id }) => _.toString(_id), lostOptions)
```

Comment:
are you sure this should be `_id` and not the node field?

Reply:
Yes, I am sure. This is really hard to find a bug, https://github.com/smartprocure/contexture-mongo/blob/5e1e876bd8d0a42f3ab8fbaa1bf0ecfafa52a76f/src/example-types/facet.js#L172 will convert the field name to id